README.md

```diff
@@ -2315,7 +2316,7 @@ The path to generate a badge is the following:
 /api/v1/endpoints/{key}/uptimes/{duration}/badge.svg
 ```
 Where:
-- `{duration}` is `30d` (alpha), `7d`, `24h` or `1h`
+- `{duration}` is `30d`, `7d`, `24h` or `1h`
```

Comment:
Fair, the 30d badges has been around for a bit now and it's fair to stay it's no longer in alpha.